benchmark/common.js

```diff
@@ -52,12 +52,13 @@ Benchmark.prototype._parseArgs = function(argv, configs) {
       // Infer the type from the config object and parse accordingly
       const isNumber = typeof configs[match[1]][0] === 'number';
       const value = isNumber ? +match[2] : match[2];
-      cliOptions[match[1]] = [value];
+      if (!cliOptions[match[1]]) cliOptions[match[1]] = [];
```

Comment:
Can we assign `match[1]` to a variable and make the block of this conditional indented on the following line instead?